File: src/main/java/com/glitch/knowledge/model/constant/StationDirection.java
package com.glitch.knowledge.model.constant;

/**
 * BIDIRECTIONAL: Station has platforms for both directions
 * <br>
 * INCREASING: One way station goes towards higher number terminus
 * <br>
 * DECREASING: One way station goes towards 0 value terminus
 * 
 * @see LineType
 */
public enum StationDirection {
    BIDIRECTIONAL,
    INCREASING,
    DECREASING
}

File: src/main/kotlin/com/glitch/knowledge/model/constant/StationDirection.kt
package com.glitch.knowledge.model.constant

/**
 * BIDIRECTIONAL: Station has platforms for both directions
 *
 * INCREASING: One way station goes towards higher number terminus
 *
 * DECREASING: One way station goes towards 0 value terminus
 * 
 * @see LineType
 */
enum class StationDirection {
    BIDIRECTIONAL,
    INCREASING,
    DECREASING,
}